proto/internal/temporal/server/api/replication/v1/message.proto

```diff
@@ -106,6 +108,7 @@ message WorkflowReplicationMessages {
     temporal.server.api.enums.v1.TaskPriority priority = 4;
 }
 
+// TODO: Deprecate this definition, it only used by the deprecated replication DLQ v1 logic
```

Comment:
this also used in replication queue?

Reply:
Replication queue uses persistence.ReplicationTaskInfo not replication.ReplicationTaskInfo 🤦 
Very confusing... I actually don't know why we have two definitions.